pom.xml

```diff
@@ -105,12 +133,11 @@
                         </goals>
                         <configuration>
                             <transformers>
-                                <transformer
-                                        implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer"/>
-                                <transformer
-                                        implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
+                                <transformer implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer"/>
```

Comment:
Please make sure you're shipping a LICENSE.txt and NOTICE.txt along with the existing artifacts. You can find several examples of how this is done in the pom.xml from our recent extensions (Netscaler is a good reference)